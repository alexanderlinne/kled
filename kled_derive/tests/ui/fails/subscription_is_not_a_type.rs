#[macro_use]
extern crate kled_derive;

#[operator(type = "flow", subscription = "struct Foo {}")]
pub struct Map<ItemOut, UnaryOp> {}

fn main() {}
