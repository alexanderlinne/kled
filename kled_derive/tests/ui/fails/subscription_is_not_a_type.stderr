error: `#[operator]` subscription must be a valid type
 --> $DIR/subscription_is_not_a_type.rs:4:42
  |
4 | #[operator(type = "flow", subscription = "struct Foo {}")]
  |                                          ^^^^^^^^^^^^^^^
